medications/README.md

```diff
@@ -0,0 +1,152 @@
+| consumable / manufacturedProduct / manufacturedMaterial / code \[@displayName]           | Mapped from `descriptionDisplay` value inside the extension within the `CodableConcept` if present.<br/>If not present then mapped from the `display` value inside the `CodableConcept`                                                                                                                                                                                 |
+| consumable / manufacturedProduct / manufacturedMaterial / code / originalText            | Mapped from `MedicationRequest.dispenseRequest.quantity.unit`                                                                                                                                                                                                                                                                                                           |
+| component \[@typeCode]                                                                   | fixed value = `COMP`                                                                                                                                                                                                                                                                                                                                                    |
+| component / ehrSupplyPrescribe / id \[@root]                                             | `MedicationRequest.id`                                                                                                                                                                                                                                                                                                                                                  | 
+| component / ehrSupplyPrescribe / code \[@code]                                           | 1. If `MedicationStatement.extension[0].valueCodeableConcept.coding[0].code` is `"prescribed-at-gp-practice"` OR `"prescribed-by-previous-practice"` then value is `394823007`.<br/>2. If `MedicationStatement.extension[0].valueCodeableConcept.coding[0].code` is `"prescribed-by-another-organisation"` then value is `394828003`.                                   |
+| component / ehrSupplyPrescribe / code \[@displayName]                                    | 1. If `MedicationStatement.extension[0].valueCodeableConcept.coding[0].code` is `"prescribed-at-gp-practice"` OR `"prescribed-by-previous-practice"` then value is `NHS Prescription`.<br/>2. If `MedicationStatement.extension[0].valueCodeableConcept.coding[0].code` is `"prescribed-by-another-organisation"` then value is `Prescription by another organisation`. |
```

Comment:
same as other comment about MedicationStatement.extension\[index]url.value

Reply:
Changed in commit.